guide/src/main/asciidoc/api_specifications.adoc

```diff
@@ -14,17 +14,17 @@ In this case, special attention should be given that the API specifications rema
 [[openapi]]
 === OpenAPI (Swagger)
 
-OpenAPI is the dominating standard for API documentation, having gained wide industry support.
+OpenAPI is the dominating standard for REST API documentation, having gained wide industry support.
 It is based on http://swagger.io/[Swagger^], which has been standardized as https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md[OpenAPI 2.0].
```

Comment:
This sentense does make it seem a little bit as if OpenAPI 2.0 is the standard?

Reply:
removed "2.0"